src/dbnode/storage/namespace/types.go

```diff
@@ -84,6 +85,10 @@ type Options interface {
 
 	// IndexOptions returns the IndexOptions.
 	IndexOptions() IndexOptions
+
+	SetSchema(schema []Schema) Options
```

Comment:
nit: comments for both methods, and mind using plural form - i.e. `SetSchemas` and `Schemas`